src/start/RealmGate.js

```diff
@@ -0,0 +1,44 @@
+/* @flow */
+import React, { PureComponent } from 'react';
+import { View, StyleSheet } from 'react-native';
+
+import connectWithActions from '../connectWithActions';
+import type { Actions } from '../types';
+import { Screen, Centerer, ZulipButton } from '../common';
+
+const componentStyles = StyleSheet.create({
+  divider: {
+    height: 20,
+  },
+});
+
+type Props = {
+  actions: Actions,
+};
+
+class RealmGate extends PureComponent<Props> {
+  props: Props;
+
+  navToRealmScreen = () => {
+    const { actions } = this.props;
+    actions.navigateToAddNewAccount();
+  };
+  navToHelp = () => {
+    const { actions } = this.props;
+    actions.navigateToHelp();
+  };
+
+  render() {
+    return (
+      <Screen title="Welcome!" centerContent padding>
+        <Centerer>
+          <ZulipButton secondary text="I have a Zulip account" onPress={this.navToRealmScreen} />
```

Comment:
As long as these methods are so simple, I think it'd actually be clearer to just inline them here. Put the `const { actions } = this.props;` at the top of `render`, and then this is still just one function -- so like
```
         <ZulipButton secondary text="I have a Zulip account"
             onPress={() => actions.navigateToAddNewAccount("")} />
```

That just reduces the number of different layers with their own names, and different steps of looking stuff up when tracing what happens.